### PR TITLE
[GH-19267] Spelling

### LINKED ISSUE #19267: Spelling errors in master branch
#### Summary
There are typos in mattermost-server

#### Steps to reproduce
How can we reproduce the issue (what version are you using?)
https://github.com/jsoref/mattermost-server/commit/4ff548f1235d2410473542bac8dd6262488c3561#commitcomment-62819286

#### Expected behavior

No typos and a ✅ 

#### Observed behavior (that appears unintentional) 
What did you see happen? Please include relevant error messages and/or screenshots.
❌ 
<details><summary>Unrecognized words (140)</summary>

```
activitated
activiy
acually
adjustement
alfter
allways
Ancilary
attachements
attachemtns
Avaliable
brakets
cannonical
categorywhich
cateory
catetories
cellspcing
Chaned
chaneld
Chanenel
Channe
comforms
commmandline
Complicance
constained
coresponding
corrensponding
couldnt't
Couns
cxt
deacivated
deactive
decprecated
depreacting
deriviative
destionation
deteminism
diemsnions
direferen
documentatin
duplcates
elipsis
emai
Emojies
emptively
Enivironment
enteprise
exaple
Exeeded
exepected
Exipry
exmaple
Failied
floag
forign
Fowarded
htp
idenitify
indentify
independant
initializaing
Initializiation
initialzed
innapropriate
invate
ivalid
Managment
markteplace
mattemost
memebership
mermbership
Metadat
metion
multipile
notexist
notifed
notoverriden
occured
ocurrs
oprphaned
orphanded
overrideable
overriden
overrite
overwites
pagnation
panicing
paramater
Patchs
Pemissions
permenent
permisssions
persistant
pontential
Posible
Predictible
prepackged
previoous
privat
promete
qouted
Receipients
recorgnizing
registeration
repoy
requrie
Resctricted
resetted
Respose
Restricions
retranslateed
retreiving
Roundtripper
Rountrippers
schemd
Schme
Searc
serch
shoudn't
should'nt
sigantures
signatrue
singup
somthing
specfified
STMP
Sucess
Suggesitons
Suscription
Sycn
syncronize
syncronizer
Sytem
theoritcally
thumbail
unknwon
updation
Usersname
webhok
Webhoook
wilrdcar
```
</details>

#### Possible fixes

- [ ] #19268
- [ ] strings
- [ ] struct fields and others

### PR DESCRIPTION
**Pull request by jsoref**
#### Summary

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

This is limited to tests

#### Ticket Link

Fixes #19267 

#### Release Note

```release-note
NONE
```

This might be split -- I'm leaving it in pieces in case there are preferences for how to split.

### CODE REVIEW

File: api4/team.go [hunk 1]
```diff
@@ -1377,12 +1377,12 @@ func inviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 func inviteGuestsToChannels(c *Context, w http.ResponseWriter, r *http.Request) {
 	graceful := r.URL.Query().Get("graceful") != ""
 	if c.App.Srv().License() == nil {
-		c.Err = model.NewAppError("Api4.InviteGuestsToChannels", "api.team.invate_guests_to_channels.license.error", nil, "", http.StatusNotImplemented)
+		c.Err = model.NewAppError("Api4.InviteGuestsToChannels", "api.team.invite_guests_to_channels.license.error", nil, "", http.StatusNotImplemented)
```
Comment (agnivade) [inline api4/team.go:1380]: These are unfortunate, but I don't think we can change the translation ids. These ids have become part of the contract and clients will hardcoded them in the code and show appropriate messages. Let's revert these and any id changes in *.json files as well.

File: model/token.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ import (
 
 const (
 	TokenSize          = 64
-	MaxTokenExipryTime = 1000 * 60 * 60 * 48 // 48 hour
+	MaxTokenExpiryTime = 1000 * 60 * 60 * 48 // 48 hour
```
Comment (agnivade) [inline model/token.go:12]: Unfortunate, but this is a breaking change we cannot do.

**Review by agnivade — COMMENTED** [review]
Thanks @jsoref. Sorry for the delay here.

**Comment by agnivade** [comment]
Btw, it appears there are some conflicts to resolve.

File: model/permission.go [hunk 3]
```diff
@@ -1101,20 +1101,20 @@ func initializePermissions() {
 	}
 	PermissionReadJobs = &Permission{
 		"read_jobs",
-		"authentication.permisssions.read_jobs.name",
-		"authentication.permisssions.read_jobs.description",
+		"authentication.permissions.read_jobs.name",
```
Comment (agnivade) [inline model/permission.go:1104]: Let's revert these as well

**Review by agnivade — COMMENTED** [review]
One last thing to go @jsoref !

File: model/permission.go (hunk 3 above)
Comment (jsoref) [reply model/permission.go]: @agnivade: dropped

**Review by agnivade — APPROVED** [review]

**Comment by agnivade** [comment]
@jsoref - Sorry this needs some changes in the enterprise code. I'll have to push this branch myself.

**Pull request closed without merging**